db/seeds.rb

```diff
@@ -141,6 +141,30 @@
   user_id: 1
 )
 
+Tag.create(
+ title: "Breakfast"
+)
+
+Tag.create(
+ title: "Lunch"
+)
+
+Tag.create(
+ title: "Dinner"
+)
+
+Tag.create(
+ title: "Snack"
+)
+
+tag1 = Tag.find(1)
+tag2 = Tag.find(2)
+
+recipe1 = Recipe.find(1)
+recipe1.tags << tag1
+recipe1.tags << tag2
```

Comment:
We could actually clean this up a bit. The `create` method returns the thing it creates, so you can say `breakfast = Tag.create(title: 'Breakfast')`. We could also do that with one of the recipes, like `recipe = Recipe.create(...)`.

Then instead of adding random tags to a random recipe, we could say `recipe.tags = [breakfast, lunch]`. That way we know which recipe has been given which tags, and it's easier to test.

Reply:
<img width="320" alt="Screen Shot 2019-10-30 at 5 26 32 PM" src="https://user-images.githubusercontent.com/39745045/67903775-791b0680-fb3a-11e9-9b70-c3005f75af1c.png">